outputs.tf

```diff
@@ -9,3 +9,7 @@ output "alb_endpoint" {
 output "alb_zone_id" {
   value = aws_alb.keycloak-load-balancer.zone_id
 }
+
+output "sqs_queue_arn" {
+  value = aws_sqs_queue.keycloak_to_alerts_concierge_user_updates.arn
```

Comment:
@ianwestcott When I `plan` I'm not seeing a value being output for this, but am I correct in assuming that it'll be spit out when `apply` is run? We'll need to know it in `devops`.

Reply:
I don't think outputs appear in the `terraform plan` output, but you should still be able to refer to this var once the module is updated in the main configuration, similar to how other outputs are already referred to, e.g. `module.keycloak-dev.sqs_queue_arn`. See references to `alb_zone_id` and `alb_endpoint` for existing examples.